README.md

```diff
@@ -182,7 +182,8 @@ After the setup is complete, you'll receive an email with your JFrog environment
 <details>
   <summary>Step 2 - Install Frogbot</summary>
 
-- [Installing Frogbot on GitHub repositories](docs/install-github.md)
+- [Installing Frogbot on GitHub repositories with GitHub Actions](docs/install-github.md)
```

Comment:
Let's also have the above "Installing Frogbot on GitHub repositories with GitHub Actions" title inside the `docs/install-github.md` file. Currently it includes "Installing Frogbot on GitHub repositories" as a title.

Reply:
Done